apps/web/pages/event-types/[type].tsx

```diff
@@ -299,6 +302,15 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     console.log(tokensList); // Just here to make sure it passes the gc hook. Can remove once actual use is made of tokensList.
 
     fetchTokens();
+
+    const generateHashedLink = (id: number) => {
+      const translator = short();
+      const seed = `${id}:${new Date().getTime()}`;
+      const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
+      return uid;
+    };
+
+    !hashedUrl && setHashedUrl(generateHashedLink(eventType.users[0].id));
```

Comment:
Generates a hash at the render time **once** so that no matter how many times the user checks/unchecks the `Generate Hash URL` setting, the link is consistent.